### DISCUSSION TITLE
How to dynamically configure BPF.

### DISCUSSION THREAD

**Discussion by ErlindS** [Security-Onion-Solutions/securityonion#11198] [General] [answered]
Normally, one would configure their BPF in security Onion 2.3 by editing the global.sls file, but what if you want to have multiple BPF configurations at different times of the day. For example, I would like to allow any traffic from 8:00-23:00, but after 23:00 I would like to block any traffic from 192.168.135.144. 
Is it possible to configure your BPFs for the example above? At the same time, is it possible to configure your BPFs without editing the ./default/ directory?

**Comment by dougburks** [comment, marked as answer]
> after 23:00 I would like to block any traffic from 192.168.135.144

To clarify, Security Onion is totally passive and so it doesn't actually block any traffic. A BPF will filter the traffic so that it is not seen by a sniffing process like Suricata, Zeek, or Stenographer.

If you want different BPF configurations at different times of day, one option might be to create cron jobs to update the config file.

**Reply by ErlindS** [reply]
Thanks for your answer and I accidentally wrote block instead of filter. An upsie from me. The solution with the cron job worked, thanks
